cmd/completion.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"errors"
+	"os"
+)
```

Comment:
Please sort imports with `go imports`. Specifically, Go standard library imports should come before third-party imports.